external/anomaly/ote/configuration.yaml

```diff
@@ -17,6 +17,7 @@ model:
     enum_name: AnomalyModel
     options:
       PADIM: padim
+      STFPM: stfpm
```

Comment:
Not sure if we need these here. As recently discussed, multiple model support is currently not available, and is not clear how backend will sort this out

Reply:
I'll remove this.